translator/translate/metrics/append_dimensions/append_dimensions_test.go

```diff
@@ -11,7 +11,7 @@ import (
 )
 
 func TestAppendDimensions(t *testing.T) {
-	e := new(appendDimensions)
+	a := new(appendDimensions)
```

Comment:
The changes to this file can be reverted.

Reply:
Also fixed it in the newest commit. Another nice catch on these though.